bin/generate-version-file.ts

```diff
@@ -0,0 +1,18 @@
+const { writeFileSync } = require("fs")
```

Comment:
Maybe a stupid question, but for my own curiosity:

could we also look this up at run time via `require('../package.json')`? If so, what are the benefits of making this file at compile time?

Reply:
@sambev ok take another look

Reply:
Ummm there might be an issue with that actually

<img width="564" alt="image" src="https://user-images.githubusercontent.com/601509/190483850-54ada197-9507-440c-8b6e-7c097d3adaee.png">

Looking into it now.

Reply:
I reverted to what I had before due to some issues with RN importing the package.json file at run time.